web/elm/src/Build/StepTree/StepTree.elm

```diff
@@ -1274,6 +1340,20 @@ tooltip model { hovered } =
             Dict.get id model.steps
                 |> Maybe.map stepDurationTooltip
 
+        HoverState.Tooltip (StepVersion _) _ ->
+            Just
+                { body =
+                    Html.div
+                        Styles.changedStepTooltip
+                        [ Html.text "view in resources page" ]
+                , attachPosition =
+                    { direction = Tooltip.Top
+                    , alignment = Tooltip.End
```

Comment:
Could we actually to `Tooltip.Start`? Might just be me, but when I was trying to use this feature, I tended to put my cursor closer to the start of the version table (near where the keys are). When the version is particularly wide (like  with the `registry-image`), the tooltip appears pretty far away

Reply:
I briefly tried this before, but there was something weird about how it calculated the tooltip width and made it longer than necessary. I also liked the way the alignment was sorta consistent in style with the image fetching and step initialization timer tooltips.

Maybe I'll just bite the bullet and figure out the values to make it `Tooltip.Middle`

Reply:
Ended up doing `Tooltip.Middle`. 

> but there was something weird about how it calculated the tooltip width and made it longer than necessary.

This happens if the version text is short enough, the tooltip overflows the side of the screen into a newline, but the tooltip box doesn't take that into consideration and is longer than it need to be.
![Screen Shot 2021-03-23 at 11 30 21 AM](https://user-images.githubusercontent.com/17577073/112172665-4825f080-8bcb-11eb-9863-75c3b017618c.png)

But I'm going to say that versions text shorter than 9 characters (I think that's around the limit) are rare enough and we don't have to worry about it.